Feature/64833 set by default to output estimate data AND 64504 AND 62722

tyndmar opened the pull request.

Commit:
64833 Set by default to output Estimate data

File changed:
asiUpdateEnv.w
- created package for data updates if currver LT 20.01.00
- in this proc, set values for NK1 CECostSave per specs in ticket

Commit:
64504 - Remove any change that turns off Credit Hold

asiUpdateEnv.w
- updated section that creates "VAL" NK1s when not present

jay-advantzware reviewed (commented):
asiUpdateEnv - Not sure why we would cycle through all VAL items and turn off?  We only want to be sure that the credit hold is on.  This seems like it would turn off all customer settings?

Commit:
62722 Mod Request - Fields in AF1 - Sales Reporting

File changed:
asiUpdateEnv.w - added load of NAICS records from .d file
builddumpfiles.p - added dump of naics to .d file list

Commit:
64504 Credit hold - Remove any change that turns off Credit Hold upon upgrade

File changed:
asiUpdateEnv.w - added code to force CreditHold NK1 to ON if current version LT 20.01.00
